src/main/java/salesforce/ui/pages/product/ProductClassicSkin.java

```diff
@@ -0,0 +1,131 @@
+
+    @FindBy(id = "IsActive_chkbox")
+    private WebElement checkboxActive;
+
+    @FindBy(id = "ProductCode_ileinner")
+    private WebElement productCode;
+
+    @FindBy(id = "Family_ileinner")
+    private WebElement family;
+
+    @FindBy(id = "Description_ileinner")
+    private WebElement description;
+
+    @FindBy(xpath = "//h2[@class='pageDescription']")
+    private WebElement headerName;
+
+    /**
+     * Gets the product name text.
+     *
+     * @return a String with the product name text.
+     */
+    public String getNameText() {
+        return fixText(getWebElementAction().getTextOnWebElement(name));
+    }
+
+    /**
+     * Gets the product product code text.
+     *
+     * @return a String with the product product code text.
+     */
+    public String getProductCodeText() {
+        return fixText(getWebElementAction().getTextOnWebElement(productCode));
+    }
+
+    /**
+     * Gets the product product family text.
+     *
+     * @return a String with the product product family text.
+     */
+    public String getProductFamilyText() {
+        return fixText(getWebElementAction().getTextOnWebElement(family));
+    }
+
+    /**
+     * Gets if product is active.
+     *
+     * @return true if it is active, false if not
+     */
+    public boolean isActive() {
+        return !checkboxActive.getAttribute("alt").contains("No");
+    }
+
+    /**
+     * Gets the description of the product.
+     *
+     * @return the description
+     */
+    public String getDescription() {
+        return getWebElementAction().getTextOnWebElement(description);
+    }
+
+    /**
+     * There is no message in the lightning skin.
+     *
+     * @return an empty message.
+     */
+    @Override
+    public String getSuccessMessage() {
+        return "";
+    }
+
+    /**
+     * Gets the product title.
+     *
+     * @return a String with the title.
+     */
+    @Override
+    public String getProductTitle() {
+        return getWebElementAction().getTextOnWebElement(headerName);
+    }
+
+    /**
+     * Gets the product's id.
+     *
+     * @return a string with the value
+     */
+    @Override
+    public String getProductId() {
+        String url = getWebElementAction().getSiteCurrentUrl();
+        return url.substring(getBaseUrlClassic().length() + 1);
+    }
+
+    /**
+     * Builds a map from the ui.
+     *
+     * @return HashMap<String, String> with the product map
+     */
+    @Override
+    public HashMap<String, String> theProductMap() {
+        HashMap<String, String> productMap = new HashMap<>();
+        productMap.put("Name", getNameText());
+        productMap.put("IsActive", isActive() + "");
+        productMap.put("ProductCode", getProductCodeText());
+        productMap.put("Family", getProductFamilyText());
+        return productMap;
+    }
+
+    /**
+     * Fixes the text if it's empty.
+     *
+     * @param text the text to fix.
+     * @return null is the text is empty, the text otherwise.
```

Comment:
method does not return null, maybe you mean empty string, and also it should be if not is.

Reply:
already fixed